ovs/extensions/healthcheck/generic_hc.py

```diff
@@ -439,3 +446,24 @@ def verify_rabbitmq(result_handler):
                 result_handler.failure('RabbitMQ has partition issues: {0}'.format(', '.join(partitions)))
         else:
             result_handler.skip('RabbitMQ is not running/active on this server!')
+
+    @staticmethod
+    @expose_to_cli(MODULE, 'recovery-domain-test', HealthCheckCLIRunner.ADDON_TYPE)
+    def check_recovery_domains(result_handler):
+        prim_domains = [domain.name for domain in DomainList.get_domains() if len(domain.storage_router_layout['regular']) >= 1]
+        for domain in DomainList.get_domains():
+            layout = domain.storage_router_layout
+            recovery = layout['recovery']
+            regular = layout['regular']
+            # Check recovery usage
+            if len(recovery) >= 1 and domain.name not in prim_domains:
+                sr_ips = str(', '.join([StorageRouter(guid).ip for guid in recovery]))
```

Comment:
`str` conversion is not necessary, since a `', '.join` will always return a string